content/en/docs/ct-logs.html

```diff
@@ -11,7 +11,15 @@
 
 <p><a href="https://www.certificate-transparency.org/what-is-ct">Certificate Transparency (CT)</a> is a system for logging and monitoring the issuance of TLS certificates. CT greatly enhances everyone's ability to monitor and study certificate issuance, and these capabilities have led to numerous improvements to the CA ecosystem and Web security. As a result, CT is rapidly becoming critical infrastructure.</p>
 
-<p>Let's Encrypt submits all certificates we issue to CT logs. We also operate a CT log ourselves, which all publicly trusted certificate authorities are welcome to submit to.</p>
+<p>Let's Encrypt submits all certificates we issue to CT logs. We also operate several CT logs ourselves, which all publicly trusted certificate authorities are welcome to submit to.
```

Comment:
s/several/two/. And at the top of the "CT Logs" section let's mention that these are two logs (Oak and Testflume), which are sharded annually.